presto-native-execution/src/test/java/com/facebook/presto/nativeworker/TestHiveWindowQueries.java

```diff
@@ -84,4 +85,101 @@ public void testRowNumber()
     {
         testRankingFunction("row_number()");
     }
+
+    protected List<String> getWindowTestQueries(Pair<String, String> functionCall)
+    {
+        ImmutableList.Builder<String> queries = ImmutableList.builder();
+        List<String> columnProjections = Arrays.asList("orderkey, commitdate, extendedprice, shipinstruct");
+        List<String> overClauses = Arrays.asList("PARTITION BY orderkey ORDER BY commitdate asc nulls first, extendedprice asc, shipinstruct desc",
```

Comment:
Do you have different partition combinations because the data types are different ?

Might be good to add a comment to explain what combinations you are testing.